internal/storage/memory.go

```diff
@@ -65,27 +66,67 @@ func newMemoryIssuerService(config Config) (*memoryIssuerService, error) {
 	return svc, nil
 }
 
-// GetByURI looks up the given issuer by URI, returning the issuer if one exists.
-func (s *memoryIssuerService) GetByURI(ctx context.Context, uri string) (*v1.Issuer, error) {
-	row := s.db.QueryRow(`SELECT id, name, uri, jwksuri, mappings FROM issuers WHERE uri = $1;`, uri)
+// Create creates an issuer.
+func (s *memoryIssuerService) Create(ctx context.Context, iss types.Issuer) (*types.Issuer, error) {
+	err := s.insertIssuer(iss)
+	if err != nil {
+		return nil, err
+	}
 
-	var iss v1.Issuer
+	return &iss, nil
+}
 
-	var mapping string
+func (s *memoryIssuerService) GetByID(ctx context.Context, id string) (*types.Issuer, error) {
+	row := s.db.QueryRow(`SELECT id, name, uri, jwksuri, mappings FROM issuers WHERE id = $1;`, id)
```

Comment:
I see a lot of manually written SQL queries in this repo. Any chance you'd use ORM?